src/config.rs

```diff
@@ -114,7 +114,7 @@ pub struct Target {
     /// Command used to launch QEMU
     pub qemu_command: Option<String>,
     /// Command to run inside virtual machine.
-    pub command: String,
+    pub command: Option<String>,
```

Comment:
Is this so you can omit the command in the config file and then override it with the cli flag? If so, maybe cleaner to use serde(default) instead: https://serde.rs/field-attrs.html#default

The `Option<T>` feels unnecessary given string can be empty and we already handle it